src/explanations/how-lendings-borrowing-aave.yaml

```diff
@@ -65,4 +65,96 @@
     After borrowing the balance of DAI tokens is 1000 DAI in my wallet.
 
     ![AAVE](https://github.com/DoDAO-io/dodao-aave-introduction-course/blob/main/images/afterborrowing.jpg?raw=true)
+
+- title: How ATokens works in Lending
+  shortTitle: Working of ATokens
```

Comment:
We can create two subheadings under this explanations. 

1) How Lending works internally with AAVE tokens
2) How Borrowing works internally with AAVE tokens